src/equations/compressible_euler_1d.jl

```diff
@@ -593,6 +593,24 @@ end
   return SVector(w1, w2, w3)
 end
 
+# Mapping and variables adapted from Hughes, Mallet, Franca (1986)
+# A new FE formulation for CFD: I. Sym. forms of the compressible Euler and NS equations 
+# and the second law of thermodynamics. https://doi.org/10.1016/0045-7825(86)90127-1
+@inline function entropy2cons(w, equations::CompressibleEulerEquations1D)
+  gamma = equations.gamma
+
+  w1,w2,w3 = w .* (gamma-1) # convert to entropy -ρ*s / (γ-1)
+  
+  wv_square   = w2^2
+  s = gamma - w1 + wv_square/(2*w3)
+
+  rho_iota     = ((gamma-1) / (-w3)^gamma)^(1/(gamma-1))*exp(-s/(gamma-1))
+  rho    = -rho_iota * w3
+  rhov   =  rho_iota * w2
+  rho_e  =  rho_iota*(1-wv_square/(2*w3))
+  return SVector(rho,rhov,rho_e)
+end
```

Comment:
```suggestion
@inline function entropy2cons(w, equations::CompressibleEulerEquations1D)
  # See Hughes, Franca, Mallet (1986) A new finite element formulation for CFD
  # [DOI: 10.1016/0045-7825(86)90127-1](https://doi.org/10.1016/0045-7825(86)90127-1)
  @unpack gamma = equations
  
  # convert to entropy `-rho * s` used by Hughes, France, Mallet (1986)
  # instead of `-rho * s / (gamma - 1)`
  V1, V2, V5 = w * (gamma - 1)
  
  # specific entropy, eq. (53)
  s = gamma - V1 + 0.5 * (V2^2) / V5
  
  # eq. (52)
  energy_internal = ((gamma - 1) / (-V5)^gamma)^inv(gamma - 1) * exp(-s / (gamma - 1))
  
  # eq. (51)
  rho    = -V5 * energy_internal
  rho_v1 = V2 * energy_internal
  rho_e  = (1 - 0.5 * (V2^2) / V5) * energy_internal
  return SVector(rho, rho_v1, rho_e)
end
```

Reply:
You can just click on "Commit suggestion" above if that's okay for you, @jlchan

Reply:
If that code is okay for you, you could update the other dimensions accordingly